config/production.json

```diff
@@ -17,6 +17,7 @@
 		"code-splitting": true,
 		"community-translator": true,
 		"desktop-promo": true,
+		"google-voucher": true,
```

Comment:
In staging, not development: https://github.com/Automattic/wp-calypso/blob/master/config/stage.json